clients/client/src/main/java/org/projectnessie/client/http/HttpClient.java

```diff
@@ -132,15 +126,6 @@ public HttpClientBuilder setBaseUri(URI baseUri) {
       return this;
     }
 
-    public String getAccept() {
-      return accept;
-    }
-
-    public HttpClientBuilder setAccept(String accept) {
```

Comment:
Do we need to worry about breaking existing clients by removing this method? I guess not since we're still at version `0.*`, but just wanted to double check.

Reply:
No, it didn't make sense to set it to a different value - otherwise all API requests would have been broken.